BPOR/BPOR.Rms/Controllers/ResearcherController.cs

```diff
@@ -170,11 +171,41 @@ public async Task<IActionResult> SubmitStudy(ResearcherStudyFormViewModel model,
         model.PortfolioSubmissionStatusOptions = context.Submitted.ToList();
         model.OutcomeOfSubmissionOptions = context.SubmissionOutcome.ToList();
 
+
+        ViewData["Title"] = getPageTitle(model.Step);
         ViewData["Step"] = model.Step;
         ViewData["TotalSteps"] = model.TotalSteps;
         return View(model);
     }
 
+    private String getPageTitle(int? step)
```

Comment:
It seems like having a different page title for each step could be a common pattern for multi-step forms.

Extend the abstract class `FormWithSteps` with an abstract property for the step name:
`public abstract string StepName { get; }`

Implement that property for the specific below case in `ResearcherStudyFormViewModel`.

Reply:
implemented property for `FormWithSteps` this also had a knock on effect for the researcher pages ( a 2 step form ).  changed the edit pages on client feedback to use the same stepNames as well.